tests/appgeo/gdal.pm

```diff
@@ -0,0 +1,25 @@
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Install gdal and perform tests
+# Maintainer: Guillaume <guillaume@opensuse.org>
+
+use strict;
+use warnings;
+use base "consoletest";
+use testapi;
+use utils;
+
+sub run {
+    my ($self) = @_;
+
+    select_console 'root-console';
```

Comment:
```suggestion
 $self->select_serial_terminal;
```
also you will need to define env. variable VIRTIO_CONSOLE=1

Reply:
and of course you need 
`use serial_terminal;`

Reply:
> also you will need to define env. variable VIRTIO_CONSOLE=1

Is it available for all workers?

Reply:
yes

Reply:
let's keep it clear - at least from my side I don't see this as must to have . It is just **nice** to have. So if with whatever reason you like to stick to 'root-console' I am fine with that

Reply:
Done